common/repo/Repo.java

```diff
@@ -75,11 +88,15 @@
   String getTextDiff(File file1, File file2) throws IOException;
 
   String getFileContents(String commitId, String path);
-  /** Get current branch name */
+
+  /** Get current branch name. */
   String currentBranch();
-  /** Retrieves the URLs for a remote (e.g. https://github.com/google/startup-os.git) */
+
+  /** Retrieves the URLs for a remote (e.g. https://github.com/google/startup-os.git). */
+  @SuppressWarnings("checkstyle:AbbreviationAsWordInName")
   String getRemoteURL();
```

Comment:
Let's `s/getRemoteURL/getRemoteUrl` instead.

Reply:
@oferb fixed in https://github.com/google/startup-os/pull/490/commits/630402f6a6a747564fc9d1b341bb508efae0591a